Documentation/CRDs/Cluster/external-cluster.md

```diff
@@ -170,6 +147,29 @@ If not installing with Helm, here are the steps to install with manifests.
 
 2. Create [common-external.yaml](https://github.com/rook/rook/blob/master/deploy/examples/common-external.yaml) and [cluster-external.yaml](https://github.com/rook/rook/blob/master/deploy/examples/cluster-external.yaml)
 
+### Import the Source Data
```

Comment:
This change doesn't affect a duplicate namespace issue. Seems like the purpose of the PR changed? How about updating the PR and commit description with the new proposed update?